docker/from-source/Debian

```diff
@@ -38,7 +42,7 @@ RUN setcap cap_fowner,cap_chown,cap_mknod,cap_sys_chroot=ep /usr/bin/coolforkit
 
 EXPOSE 9980
 
-# switch to cool user (use numeric user id to be compatible with Kubernetes Pod Security Policies)
-USER 101
+ # switch to cool user (use numeric user id to be compatible with Kubernetes Pod Security Policies)
```

Comment:
I actually tested it by creating another docker file, so while copy-pasting it crept in. I thought it would create any problem so I just left it like that